apps/api/src/app/message-template/usecases/update-message-template/update-message-template.usecase.ts

```diff
@@ -33,7 +33,7 @@ export class UpdateMessageTemplate {
       updatePayload.name = command.name;
     }
 
-    if (command.content !== null) {
+    if (command.content !== null || command.content !== undefined) {
```

Comment:
Related to a different [PR](https://github.com/novuhq/novu/pull/5295#discussion_r1523214826), added it here after a comment there